app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,56 @@
+class CategoriesController < ApplicationController
+
+  before_action :find_item,      only:[:show,:edit,:destroy]
+
+  def index
+    @categories = Category.all
+    @category = Category.all
+  end
+
+  def new
+    @category = Category.new
+  end
+
+  def edit
+
+  end
+
+  def update
+    @category = Category.where(id: params[:id]).first
```

Comment:
```suggestion
    @category = Category.where(id: params[:id]).first
```

ну почему тут `first`? сделай нормально с `find`